docassemble/EFSPIntegration/data/questions/case_search.yml

```diff
@@ -16,7 +16,10 @@ code: |
     somebody.name.first
     court_id    
     get_cases_response
-    case_tracking_id = case_tracking_choice
+    if case_tracking_choice is None:
+      case_tracking_id
```

Comment:
I think this sets the tracking id (which is what I've been calling the Tyler-internal GUIDs for cases / filings) so we can continue on in the Any filing interview, right? If so, I think it means that the user can't find the case, and is likely non-indexed meaning there isn't actually a tracking id. They should have a docket number, but that'll be a different question / variable.

Reply:
agreed, this is just slightly better than setting the case_tracking_id to None. We still need a full fix for it